parser.y

```diff
@@ -6443,6 +6443,10 @@ FunctionCallGeneric:
 	{
 		$$ = &ast.FuncCallExpr{FnName: model.NewCIStr($1), Args: $3.([]ast.ExprNode)}
 	}
+|	Identifier '.' Identifier '(' ExpressionListOpt ')'
+	{
+		$$ = &ast.FuncCallExpr{FnName: model.NewCIStr($1 + "." + $3), Args: $5.([]ast.ExprNode)}
```

Comment:
This is inappropriate to use `$1 + "." + $3` as the function name, since each identifier itself can contain a dot. 

Counter example on MySQL:

```sql
create schema s;
use s;
create function `s.a` () returns int deterministic return 1;
create function `a` () returns int deterministic return 2;

select `s`.`a`();  -- value is 2
select `s.a`();    -- value is 1 
```